src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -118,6 +118,8 @@ def load_arguments(self, _):
                        help='Encryption type. EncryptionAtRestWithPlatformKey: Disk is encrypted with XStore managed key at rest. It is the default encryption type. EncryptionAtRestWithCustomerKey: Disk is encrypted with Customer managed key at rest.')
             c.argument('disk_encryption_set', min_api='2019-07-01', help='Name or ID of disk encryption set that is used to encrypt the disk.')
             c.argument('location', help='Location. Values from: `az account list-locations`. You can configure the default location using `az configure --defaults location=<location>`. If location is not specified and no default location specified, location will be automatically set as same as the resource group.')
+            c.argument('network_access_policy', min_api='2020-05-01', help='Policy for accessing the disk via network. Allowed values: AllowAll, AllowPrivate, DenyAll.', arg_type=get_enum_type(self.get_models('NetworkAccessPolicy')))
```

Comment:
Will there be duplicate `Allowed values`?

Reply:
sorry, what does `dulpicate Allowed values` mean here?

Reply:
I thought `arg_type=get_enum_type(self.get_models('NetworkAccessPolicy'))` would append `Allowed values` automatically.

Reply:
Oh, I see. It seems no duplications here, just works normally.